src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -281,6 +317,18 @@ def perform_post_processing(
             obs_data = do_collide_ssns(obs_data, maps["simulant_id"]["ssn"], randomness)
 
         obs_data = obs_data[list(FINAL_OBSERVERS[observer])]
+
+        if public_sample:
+            address_part_values = {
+                "city": "Anytown",
+                "state": "USA",
```

Comment:
I'd recommend the "state" be "US" so that it matches the same two character format real states will have.